Create "Secure Command & Control" solutions page (#6312)
mattermost/docs: source/about/secure-command-and-control.rst

Secure command and control
============================

Mattermost's Secure Command and Control solution is designed to provide fast and secure mobile communications for technical teams. 

> It addresses the pain points of data leakage to unsecured channels like SMS and WhatsApp, and the lack of control and compliance from vendor-hosted mobile applications that leaves technical teams struggling to collaborate securely.

Reviewer: ```suggestion
It addresses the pain points of data leakage to unsecured channels, like SMS and WhatsApp, and the lack of control and compliance from vendor-hosted mobile applications that leaves technical teams struggling to collaborate securely.
```

The solution provides easy-to-use collaboration for technical teams, including team messaging, file sharing, audio and screen-share, and process automation directly from your mobile device on iOS or Android. The solution is specifically designed to address the challenges of enterprise-scale secure mobile communications.

Features include:

 * `Private mobile communications </deploy/mobile-overview.html>`__ to provide secure mobile apps via encryption keys and certifications with custom mobile applications.

>  * `Custom push notification service <https://docs.mattermost.com/deploy/mobile-hpns.html>`_ to ensure notifications are not intercepted or leaked to unsecured channels by transmitting push notifications within your private network.

Reviewer: ```suggestion
 * `Custom push notification service </deploy/mobile-hpns.html>`__ to ensure notifications are not intercepted or leaked to unsecured channels by transmitting push notifications within your private network.
```

>  * `Remote user deactivation with data purge <https://docs.mattermost.com/deploy/client-side-data.html#mobile-app-experience>`_ to mitigate sensitive data from being stored on mobile devices after a user departs the organization.

Reviewer: ```suggestion
 * `Remote user deactivation with data purge </deploy/client-side-data.html#mobile-app-experience>`__ to mitigate sensitive data from being stored on mobile devices after a user departs the organization.
```

 * (Professional & Enterprise) `Hosted push notifications service (HPNS) <https://docs.mattermost.com/deploy/mobile-hpns.html#hosted-push-notifications-service-hpns>`_ for production-level uptime SLAs and encrypted TLS connections between HPNS and Apple Push Notification Services, between HPNS and Google’s Firebase Cloud Messaging Service, and between HPNS and your Mattermost Server.

>  * (Professional & Enterprise) Mobile single-sign-on with `SAML <https://docs.mattermost.com/onboard/sso-saml.html>`_, `AD/LDAP <https://docs.mattermost.com/onboard/ad-ldap.html>`_, `OpenID Connect <https://docs.mattermost.com/onboard/sso-openidconnect.html>`_, `Google <https://docs.mattermost.com/onboard/sso-google.html>`_, and `Office365 <https://docs.mattermost.com/onboard/sso-office.html>`_ for centralized identity management and automatic account provisioning for fast and secure access to mobile applications.

Reviewer: ```suggestion
 * (Professional & Enterprise) Mobile single-sign-on with `SAML </onboard/sso-saml.html>`__, `AD/LDAP </onboard/ad-ldap.html>`__, `OpenID Connect </onboard/sso-openidconnect.html>`__, `Google </onboard/sso-google.html>`__, and `Office365 </onboard/sso-office.html>`__ for centralized identity management and automatic account provisioning for fast and secure access to mobile applications.
```

 * (Enterprise) `ID-only push notifications <https://docs.mattermost.com/deploy/mobile-hpns.html#id-only-push-notifications>`_ to remove the need to pass clear text notifications through Apple and Google’s mobile notification relays.

>  * (Enterprise) `Enterprise Mobility Management (EMM) <https://docs.mattermost.com/deploy/deploy-mobile-apps-using-emm-provider.html>`_ to manage secure mobile endpoints with a managed app configuration (`via AppConfig <https://docs.mattermost.com/deploy/mobile-appconfig.html>`_), including AirWatch, Blackberry Dynamics and MobileIron.

Reviewer: ```suggestion
 * (Enterprise) `Enterprise Mobility Management (EMM) </deploy/deploy-mobile-apps-using-emm-provider.html>`__ to manage secure mobile endpoints with a managed app configuration (`via AppConfig </deploy/mobile-appconfig.html>`__), such as AirWatch.
```

For an example case study, learn why `The U.S. Department of Defense relies on Mattermost for Secure Command and Control across devices <https://mattermost.com/customers/us-department-of-defense/>`_, including flight crews who can now securely access documents from mobile devices anywhere in the world. 

> `Contact us <https://mattermost.com/contact-sales/>`_ to learn more about this solution and to discuss if it’s the right one for you.

Reviewer: ```suggestion
`Contact us <https://mattermost.com/contact-sales/>`_ to learn more about this solution and to discuss whether it’s the right one for you.
```
